api_master.js

```diff
@@ -21,6 +21,14 @@ function getCreatedQuizQuestions(quizId) {
   return handleGetRequests(`/getcreatedquizquestions/${quizId}`);
 }
 
+function sendFriendEmail(user, friend, friendEmail, message) {
+  return handleGetRequests(`/email/${user}/${friend}/${friendEmail}/${message}`);
```

Comment:
I don't think we need to change, but exposing the API as an endpoint like this will let people spoof emails from us and send crazy messages to people.